### ISSUE TITLE
Exclude finds

### ISSUE THREAD

**Issue by travels4jack** [mirsch/lab2gpx#4]
Hi there!
Amazing job. Most of the features work well
However, I have a slight issue with my own finds that 5for some only, not all) apprae on the gpx file.

Should I cut and paste the entire code source page ? 
Any tip to fix this ?
Many thanks

**Comment by mirsch** [comment]
I've upload a new version 2h ago, which should fix this issue.
please try again and let me know

**Comment by travels4jack** [comment]
Hi there !
Thanks for the expedite fix. Seems to work right now.
Great feature thanks !

**Comment by PlasmaWave** [comment]
MY COMPLETED LABS ARE NOT EXCLUDED WHEN RUNNIG FROM iPHONE

When running from a Windows10 PC browser, cutting & pasting the source code works perfectly and my completed Labs are excluded from the gpx.

However working from my iPhone7+ pasting above source code via a text file, OR using a dedicated app "HTML Viewer" (copy&paste), then my completed Labs are not excluded. I see your superb "app" being most useful when out & about, hence via a mobile phone.

Any thoughts, i.e. do I need to transfer the source code differently? Anyone else experiencing same issue? I cannot see how to view the source code directly from within iPhone's Safari or Mb Chrome/Firefox, hence obtaining the source code via some other route.

Many thanks.

**Comment by mirsch** [comment]
Most of the HTML source viewer apps will not work because you are not logged in and you get the source of the login form instead the list of your founds.
I use a bookmarklet in Safari to view the source, this works well. (However, I wish there was a possibility to select the entire text at once)

**Comment by mirsch** [comment]
I've created a bookmarklet, which creates a textarea with the source and select the entire text.
https://gist.github.com/mirsch/41e2a152ba2c1e50360658925ed82b47

**Comment by mirsch** [comment]
saving the source as file on pc should work but you need to make sure the source is not modified. e.g. you should use notepad instead of word.

**Comment by PlasmaWave** [comment]
Many thanks ... I opened the link but no idea what next and how to put that onto the phone. Sorry for being slow but I hadn't even heard of bookmarklets until you mentioned it earlier. Am trying to read up via internet but not understanding how to use it or your link. Any chance of an explanation for simpletons? Maybe a 'step by step' if you have the time.

BTW I confirmed what you said about the HTML viewer app ... my Lab data wasn't in the source code it generated. 

Regarding the source code from my PC browser, I did paste it into Notepad and saved to Dropbox, then opened the file from the phone and copied & pasted into your script box. My completed Labs were not however excluded ... I rechecked the code afterwards and it appeared to still have all the Lab stage names ok.
